bot/utils/parameter_value_generator.py

```diff
@@ -28,22 +31,29 @@ def parameter_value_generator(
         lower_bound: numerical
         upper_bound: numerical
     Returns:
-        param_value: numerical
+        params: pybamm.ParameterValues
```

Comment:
This now returns `params` and `varied_value`, so you could update the docs or instead just return `params`. Then in the other files you could do `params[param_to_vary]` anytime you need the value. My personal preference would be to just return `params` since you always know `param_to_vary` anyway so it seems unnecessary to return `varied_value` too. No big deal though, so keep whichever you prefer  :)

Reply:
Yes, returning only params this will be cleaner